### PR TITLE
Portfolio: mobile version (2nd part)

### PR DESCRIPTION
**Pull request by fbclh**
* Finish all the HTML and CSS for the second part of the project.
* _Create this new v2 branch base on the working-in-progress branch which already has been approved, while I was waiting for it to be approve._

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -69,6 +69,187 @@ <h1 class="header">
           </li>
         </ul>
       </section>
+
+      <!-- Cards -->
+      <section class="portfolio-cards">
+        <div class="card one">
+          <div class="card-img">
+            <img src="img/snapshoot1.png" alt="" />
+          </div>
```
Comment (keneogu) [inline index.html:78]: - [ ] From the Figma design, your card-img should be a background image, not `` tag. please fix it.

File: index.html [hunk 2]
```diff
@@ -69,6 +69,187 @@ <h1 class="header">
...
+            </ul>
+            <button class="btn card" type="button">See Projects</button>
+          </div>
+        </div>
+      </section>
+      <section class="portfolio-cards">
+        <div class="card two">
+          <div class="card-img">
+            <img src="img/snapshoot2.png" alt="" />
+          </div>
```
Comment (keneogu) [inline index.html:105]: - [ ] From the Figma design, your card-img should be a background image, not `` tag. Add a background image not an actual image with the `<img` tag.

File: index.html [hunk 3]
```diff
@@ -69,6 +69,187 @@ <h1 class="header">
...
+            </ul>
+            <button class="btn card" type="button">See Projects</button>
+          </div>
+        </div>
+      </section>
+      <section class="portfolio-cards">
+        <div class="card three">
+          <div class="card-img">
+            <img src="img/snapshoot3.png" alt="" />
+          </div>
```
Comment (keneogu) [inline index.html:132]: - [ ] From the Figma design, your card-img should be a background image, not `` tag. Add a background image not an actual image with the `<img` tag.

File: index.html [hunk 4]
```diff
@@ -69,6 +69,187 @@ <h1 class="header">
...
+            </ul>
+            <button class="btn card" type="button">See Projects</button>
+          </div>
+        </div>
+      </section>
+      <section class="portfolio-cards">
+        <div class="card four">
+          <div class="card-img">
+            <img src="img/snapshoot4.png" alt="" />
+          </div>
```
Comment (keneogu) [inline index.html:159]: - [ ] From the Figma design, your card-img should be a background image, not `` tag. Add a background image not an actual image with the `<img` tag.

File: index.html [hunk 5]
```diff
@@ -69,6 +69,187 @@ <h1 class="header">
...
+      <!-- About -->
+      <section class="about about-me">
+        <h1 class="header">About<br />Myself</h1>
+        <p class="about-paragraph">
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hestiate to contact
+          me.
+        </p>
+        <p class="links-paragraph">LET’S CONNECT</p>
```
Comment (keneogu) [inline index.html:192]: - [ ] Let's connect is a title, always make use of `<hx>` tags for titles and `<p>` tag for black of texts(paragraphs). Make use `<hx>` tag for this.

**Review by keneogu — CHANGES_REQUESTED** [review]
Hi @fbclh , 

Good job so far!

- Great commit history!
- You passed all the Linters!
- Professional pull request!
- Using Flexbox

There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️

- [ ] Your project background color should be the same as the Figma template design been given
your design
bac
Figma design
back1

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------

**Comment by fbclh** [comment]
Hello there thanks for your feedback, I have made the necessary changes  based on your feedback.

* Change the background color to match the design
* Change the p tag for h3 tag
* (Regarding the card-img  for the cards **they are part of the content** of the page and are not for styling and design purpose etc..., as such it should be that way. If you need a refresher: https://www.w3docs.com/snippets/html/when-to-use-html-img-tag-and-css-background-image-property.html)

**Review by andersonlebon — APPROVED** [review]
Hi @fbclh 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- You  are following GitHub actions :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Your README file is now professional :heavy_check_mark: 
- Your design is looking good :+1: :heavy_check_mark: 

## Optional suggestions
- [ ] Kindly Check the comments under the review._
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**